### DISCUSSION TITLE
Publico objetivo

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#105] [Q&A]
Del Paso 4: Selecciona de dos a cinco identificadores. ¿A que se refiere con identificadores? que de cada uno de los datos que puso en los primeros 3 pasos se van a elegir entre 2 a 5 de cada uno. O en general de los 3 pasos se sacan entre 2 y 5.

**Comment by doguedogue** [comment]
No soy experto en el tema, pero me imagino que cada uno de los subgrupos, que opinas?

Paso 1: Recopilar datos demográficos (6 identificadores)
edad, género, ingresos, nivel educativo, estado civil y ocupación

Paso 2: Recopilar datos de ubicación (4 o 5 identificadores)
barrio, ciudad, estado, país o región

Paso 3: Recopilar datos psicográficos (4 identificadores)
actitudes, opiniones, intereses y actividades

**Reply by GloriaCoralCerecedo** [reply]
Osea como:

1. Género, edad, ocupación 
2. Ciudad, estado
3. actitudes

Así por ejemplo(?)

Igual los datos que se ponen son tipo:

1. Edad: 18-35
2. Escolaridad: Licenciatura, maestría, doctorado etc.

O solo es 1 o un rango de cada uno (?)
